### ISSUE TITLE
pre-revprop-change hook not working!

### ISSUE THREAD

**Issue by starofrainnight** [MarvAmBass/docker-subversion#2]
I have pulled latest image and success configure the image on my Ubuntu 14.04 server. 

Now I could check out / commit to my projects through TortoiseSVN on Windows XP SP3, but I can't change the log message (I change the log message by "Edit log message" function in TortoiseSVN) while I want to do it. It show up an error message that 

```
DAV requests failed; it's possible that the repository's pre-revpro-change hook either failed or is non-existent 
Revprop change blocked by pre-revprop-change hook (exit code 255) with no output
```

I already checked the hook script's properties, it have executable attribute and belongs to www-data, and it works if I start our subversion daemon not that image.

Is I missing something?

**Comment by MarvAmBass** [comment]
actually I don't get your problem or have any idea to fix this.

there is no subversion daemon running inside the container just a webserver maybe thats the problem

**Comment by starofrainnight** [comment]
Have you tried triggered any subversion repository hooks and with success result? 

I have a server build on top of centos 6.6 also just a webserver and without any subversion daemon, but it works well with hooks, I could change the log message and passed the pre-revprop-change hook.

**Comment by starofrainnight** [comment]
I'm trying to find out why, hope later I could give more information here ..

**Comment by starofrainnight** [comment]
I found the reason why the hooks can't works!

I tried to use "docker exec" to launch a shell into the runtime container, and found that the hooks scripts run with errors:

`bash: ./pre-revprop-change: /bin/sh^M: bad interpreter: No such file or directory`

I found that all the hooks are in DOS format (with CRLF line endings), that cause scripts failed to run as shell scripts.

Those scripts be mis-converted to DOS format during a transfer procedure from my server to another server.

So, after I convert all scripts from DOS format to UNIX, all thing works!
